CodingChallenges/CC_115_Snake_Game_Redux/P5/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+
+	<title>Snake Game Redux</title>
+
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.6.1/p5.min.js"></script>
```

Comment:
```suggestion
	<script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.7.2/p5.min.js"></script>
```

does 0.7.2 work ok with the code?

Reply:
Yes. I copied the index.html from an earlier challenge that used 0.6.1. 
Thanks @shiffman!